docs/releases/v2018.2.3.rst

```diff
@@ -0,0 +1,86 @@
+Gluon 2018.2.3
+==============
+
+Added hardware support
+~~~~~~~~~~~~~~~~~~~~~~
+
+ar71xx-generic
+^^^^^^^^^^^^^^
+
+* TP-Link
+
+  - CPE210 v3
+  
+ar71xx-nand
+^^^^^^^^^^^
+
+* Aerohive
+
+  - HiveAP 121
+
+mcp85xx-p1020
+^^^^^^^^^^^^^
+
+* Aerohive
+
+  - HiveAP 330
+
+ramips-mt76x8
+^^^^^^^^^^^^^
+
+* TP-Link
+
+  - TL-MR3420 v5 [#noibss]_
+  
+.. [#noibss]
+  Device or target does not support AP+IBSS mode: This device or target will not be built
+  when *GLUON_WLAN_MESH* is set to ``ibss``.
+
+
+Bugfixes
+~~~~~~~~
+
+* fix for passwordless ssh when gluon-authorized-keys was used without gluon-setup-mode
+
+* fix for broken ingress traffic-shaping
+
+* fix packages for GL.iNet AR750 (relevant for IBSS mesh)
+
+* fix primary mac selection on TP-Link Archer C25 v1
+
+* fix for bootloader image generation for AVM Fritz!Box 4040
+
+
+Other changes
+~~~~~~~~~~~~~
+
+* Linux kernel has been updated to 4.9.188 or 4.14.137, depending on the target
+
+
```

Comment:
should 2280cb3a09ea4eb2b4c68154f60fd7d7cc9dee80 get a mention?

Reply:
it has, please see bugfixes section :)

Reply:
Danke. Ich hör ann für heute auf mit reviews.